File: 151001/Shlyk/SpringCalculator/spring-calculator-rest/src/main/java/by/bsuir/poit/dc/rest/api/dto/response/UserDto.java
package by.bsuir.poit.dc.rest.api.dto.response;

import com.baidu.bjf.remoting.protobuf.annotation.ProtobufClass;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 * DTO for {@link by.bsuir.poit.dc.rest.model.User}
 */
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@ProtobufClass
public class UserDto {
    private Long id;
    private String login;
    private String firstname;
    private String lastname;
}

File: 151004/Pyshny/RV/RV/Views/DTO/UserDTO.cs
using System.ComponentModel.DataAnnotations;

namespace RV.Views.DTO
{
    public class UserDTO
    {
        [Required]
        public int id { get; set; }
        [Required]
        public string login { get; set; }
        [Required]
        public string password { get; set; }
        [Required]
        public string firstname { get; set; }
        [Required]
        public string lastname { get; set; }
    }
}
